examples/dt/uni04delta-ipv6/control-plane/service-values.yaml

```diff
@@ -35,20 +35,28 @@ data:
     customServiceConfig: |
       [DEFAULT]
       debug = True
-      enabled_backends = default_backend:rbd
+      enabled_backends = default_backend:s3
 
       [glance_store]
       default_backend = default_backend
 
       [default_backend]
-      rbd_store_ceph_conf = /etc/ceph/ceph.conf
-      store_description = "RBD backend"
-      rbd_store_pool = images
-      rbd_store_user = openstack
+      s3_store_create_bucket_on_put = True
+      s3_store_bucket_url_format = "path"
+    customServiceConfigSecrets:
+      - s3glance
     glanceAPIs:
       default:
         replicas: 3
 
+  s3glance.conf : |
```

Comment:
Do we need this? I see you create it via ci-framework, so in theory we do not require this definition in architecture, but I might be wrong [1].

[1] https://github.com/openstack-k8s-operators/ci-framework/pull/3169